app/angular/src/server/framework-preset-angular-cli.ts

```diff
@@ -124,3 +130,65 @@ async function getBuilderOptions(
 
   return builderOptions;
 }
+
+/**
+ * Get options from legacy way
+ * /!\ This is only for backward compatibility and wild be removed on Storybook 7.0
+ * only work for angular.json with [defaultProject].build or "storybook.build" config
+ */
+async function getLegacyDefaultBuildOptions(options: PresetOptions) {
+  if (options.angularBrowserTarget !== undefined) {
+    // Not use legacy way with builder (`angularBrowserTarget` is defined or null with builder and undefined without)
+    return {};
+  }
+
+  logger.info(`=> TODO ~ legacy way to get default options`);
+  logger.info(`=> TODO ~ Add deprecation warning and ex for builder use ? `);
```

Comment:
lets remove those and merge the PR to hotfix the current issue and improve the situation afterwards
```suggestion
```

Reply:
Typo here: "and wild be removed on Storybook 7.0"